### PR TITLE
To solve question #20722

### PR DESCRIPTION
**Pull request by kebenxiaoming**
website:
https://github.com/owncloud/core/issues/20722

### CODE REVIEW

File: apps/files/js/favoritesfilelist.js [hunk 1]
```diff
@@ -64,6 +64,377 @@ $(document).ready(function() {
 				// no op because it doesn't have
 				// storage info like free space / used space
 			},
+                        _renderRow: function(fileData, options) {
+				options = options || {};
+				fileData.name = fileData.name;
+				fileData.displayName = fileData.name + '.d' + Math.floor(fileData.mtime / 1000);
+			return OCA.Files.FileList.prototype._renderRow.call(this, fileData, options);
+		        },
+		        _createRow: function(fileData, options) {
```
Comment (PVince81) [inline apps/files/js/favoritesfilelist.js]: No need to copy createRow, _renderRow is enough.

Also on line 70 you need to combine the file name with the file id

**Comment by kebenxiaoming** [comment]
No,I made a mistake,sorry,Please close this first.

**Comment by PVince81** [comment]
@kebenxiaoming did you copy createRow as is or did you make specific changes to make it work ?

**Comment by kebenxiaoming** [comment]
I have override createRow and also maked some changes to make id work.

**Comment by PVince81** [comment]
Ok thanks. I think it's not good to override it completely because it duplicates a lot of code and makes it hard to maintain. Maybe we need another approach.
I'll have a look at this later.

**Comment by kebenxiaoming** [comment]
@PVince81 Yes,we should change this more simply.

**Comment by PVince81** [comment]
@kebenxiaoming if you don't mind I'd like to close this PR and revisit the fixing approach when I find some time to handle the issue.

Unless you want to give this another try at simplifiying the code ? Note that we need a statement regarding the license, see https://github.com/owncloud/core/pull/20754#issuecomment-186721538

**Comment by kebenxiaoming** [comment]
@PVince81 Sorry,I don't have enough time to do this recently.And I think that if we want to fix this simply,perhaps we need to modify the core javascript  file:"file.js".

**Comment by PVince81** [comment]
@kebenxiaoming this is totally fine. I'll redo the PR with a simpler approach later.
Thanks!

**Pull request closed without merging**